nms/app/packages/magmalte/app/state/lte/EquipmentState.js

```diff
@@ -195,6 +199,19 @@ export async function SetEnodebState(props: EnodebStateProps) {
       const prevEnbSt = enbInfo[key].enb_state;
       setEnbInfo({...enbInfo, [key]: {enb_state: prevEnbSt, enb: value.enb}});
     }
+    const newEnb = await MagmaV1API.getLteByNetworkIdEnodebsByEnodebSerial({
+      networkId: networkId,
+      enodebSerial: key,
+    });
```

Comment:
Good point. Mostly it shouldn't. But I have found some cases where the config entity and status entity differs(for e.g. mutable_gateway and gateway). In future, if we do diverge it will keep our status consistent, and also in cases where the backend reacts to the config and sets something immediately, this may differ.  Hence doing a get call here.  I'm being defensive here and added similar code in most of the state APIs.

Reply:
Could a comment/TODO be added?